python/tests/test_drawing.py

```diff
@@ -1918,6 +1918,34 @@ def test_known_svg_ts(self):
             expected_svg = file.read()
         self.assertXmlEquivalentOutputs(svg, expected_svg)
 
+    def test_known_svg_ts_plain(self):
+        """
+        Plain style, with no background shading and a variable X axis
+        """
+        ts = self.get_simple_ts()
+        svg = ts.draw_svg(x_scale="treewise")
+        # Prettify the SVG code for easy inspection
+        svg = xml.dom.minidom.parseString(svg).toprettyxml()
+        svg_fn = pathlib.Path(__file__).parent / "data" / "svg" / "ts_plain.svg"
+        self.verify_basic_svg(svg, width=200 * ts.num_trees)
+        with open(svg_fn, "rb") as file:
+            expected_svg = file.read()
+        self.assertXmlEquivalentOutputs(svg, expected_svg)
+
+    def test_known_svg_ts_with_xlabel(self):
+        """
+        Style with X axis label
+        """
+        ts = self.get_simple_ts()
+        svg = ts.draw_svg(x_label="genomic position (bp)")
+        # Prettify the SVG code for easy inspection
+        svg = xml.dom.minidom.parseString(svg).toprettyxml()
+        svg_fn = pathlib.Path(__file__).parent / "data" / "svg" / "ts_xlabel.svg"
+        self.verify_basic_svg(svg, width=200 * ts.num_trees)
```

Comment:
Be nice to test that the specified x_label text is in the xml.

Reply:
Doh: I made a doc change then forgot to pass the parameter through. Makes the point that I should indeed check for the string in the SVG. Will update.